src/hooks/AllPaymentHooks.res

```diff
@@ -355,11 +355,17 @@ let useRedirectHook = () => {
   let (allApiData, setAllApiData) = React.useContext(AllApiDataContext.allApiDataContext)
   let redirectioBrowserHook = useBrowserHook()
   let retrievePayment = useRetrieveHook()
+  let handleSuccessFailure = useHandleSuccessFailure()
   let apiLogWrapper = LoggerHook.useApiLogWrapper()
   let logger = LoggerHook.useLoggerHook()
   let baseUrl = GlobalHooks.useGetBaseUrl()()
   let handleNativeThreeDS = NetceteraThreeDsHooks.useExternalThreeDs()
 
+  let getActionType = (nextActionObj: option<PaymentConfirmTypes.nextAction>) => {
```

Comment:
Move this function to an appropriate utils file ( for e.g. PaymentUtils)

Reply:
moved 👍